### ISSUE TITLE
Session expiration could be handled better

### ISSUE THREAD

**Issue by Chemrat** [go-shiori/shiori-web-ext#12]
Having to go to shiori page and re-login is pretty inconvenient, at least redirect there automatically, give a login popup or give ability to save credentials to re-login automatically

**Comment by fmartingr** [comment]
Hey @axelsimon, thanks for the detailed issues you found. My main focus right now is on the new API, which I hope it solve some of the server related issues you're experiencing at the moment: https://github.com/go-shiori/shiori/pull/497. Main change that extension will benefit from is the use of JWTs for sessions, which are not stored in memory, meaning that at least a server restart won't "expire" them and that expiration would be either configurable or unlimited, depending on the circumstances.

**Comment by axelsimon** [comment]
I didn't answer back in March, but good to hear. And congrats, I see https://github.com/go-shiori/shiori/pull/497 is now done :)

Does this mean this might see some progress?

**Comment by fmartingr** [comment]
> I didn't answer back in March, but good to hear. And congrats, I see [go-shiori/shiori#497](https://github.com/go-shiori/shiori/pull/497) is now done :)
> 
> Does this mean this might see some progress?

Yes, JWT sessions are landing in v1.6.0 (next release), once that's released we need to release a new version of the plugin too.
